src/connectors/idagio.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const symphonySelector = '.player-PlayerInfo__infoEl--2jhHY span:nth-child(3) span';
+const movementSelector = '.player-PlayerInfo__infoEl--2jhHY span span span';
+const directorSelector = '.player-PlayerInfo__recordingInfo--15VMv>span:first-child span';
+const recordingInfoSelector = '.player-PlayerInfo__recordingInfo--15VMv';
+const pauseButtonSelector = '.player-PlayerControls__btn--1r-vy:nth-child(2) .util-IconLabel__component--3Uitr span';
+
+Connector.playerSelector = '.player-PlayerBar__bar--2yos_';
+
+Connector.artistSelector = '.player-PlayerInfo__infoEl--2jhHY span:first-child';
+
+Connector.getTrack = () => getCurrentTrack();
+
+Connector.getAlbum = () => getCurrentSymphony();
+
+Connector.currentTimeSelector = '.player-PlayerProgress__progress--2F0qB>span';
+
+Connector.durationSelector = '.player-PlayerProgress__timeTotal--3aHlj span';
+
+Connector.isPlaying = () => document.querySelector(recordingInfoSelector).innerHTML !== 'Sponsor message' && document.querySelector(pauseButtonSelector).innerHTML === 'Pause';
```

Comment:
Skipping ads and sponsor messages should be done by using `Connector.isScrobblingAllowed` function.

By the way, is there a reason to use `innerHTML` here? If you want to get a text from an element, using `Util.getTextFromSelectors` is recommended.

Reply:
Ah i was thinking of avoiding some freak accident because it uses the same element for multiple things, but i dont think it should be an issue on second thought